jira/resources.py

```diff
@@ -899,7 +900,13 @@ def delete(self, moveFixIssuesTo=None, moveAffectedIssuesTo=None):
         return super(Version, self).delete(params)
 
     def update(self, **args):
-        """Update this project version from the server. It is prior used to archive versions."""
+        """
+        Update this project version from the server. It is prior used to archive versions.
+
+        link to atlassian restapi `documentation`_.
```

Comment:
As this docstring will be user facing in the docstring, we can make it a little nicer ;)

```suggestion
        Refer to Atlassian  REST API `documentation`_ for all possible values.
```